ups/setup_deps

```diff
@@ -59,27 +59,31 @@ set_ msg5='ERROR: setup of required products has failed'
 
 echo ----------- check this block for errors -----------------------
 set_ setup_fail="false"
-set_ cetb=` grep only_for_build $CETPKG_SOURCE/ups/product_deps | grep cetbuildtools | awk '{ print $2 }' `
-set_ cetv=` grep only_for_build $CETPKG_SOURCE/ups/product_deps | grep cetbuildtools | awk '{ print $3 }' `
+set_ exit_now="false"
+set_ cetb=` grep -e '^[ \t]*cetbuildtools' $CETPKG_SOURCE/ups/product_deps | grep  only_for_build| awk '{ print $1 }' `
+set_ cetv=` grep -e '^[ \t]*cetbuildtools' $CETPKG_SOURCE/ups/product_deps | grep  only_for_build| awk '{ print $2 }' `
```

Comment:
What's going on in this file? Presumably it is autogenerated? Did something update it?

Reply:
Yep---when I used the `setup_for_development` script, it gave me some output that told me this (and `setup_for_development` itself) were out of date, and that I should use some newer templates.  It gave me instructions, and I followed them.  (see the commit note on 5ff85adf.)